akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/PersistentBehaviorImpl.scala

```diff
@@ -16,7 +16,7 @@ import akka.util.ConstantFun
 @InternalApi
 private[akka] final case class PersistentBehaviorImpl[Command, Event, State](
   persistenceId:     String,
-  initialState:      State,
+  emptyBehavior:     State,
```

Comment:
`emptyState` here as well I presume?

Reply:
Oops, thanks for finding this. Fixed in the new commit.